src/fxp.d.ts

```diff
@@ -86,3 +86,5 @@ export class XMLBuilder {
   constructor(options: XmlBuilderOptionsOptional);
   build(jObj: any): any;
 }
+
+export function format(xmlString: string): string;
```

Comment:
I haven't written any documentation yet, I am waiting until we agree on the desired API.

What is your recommendation? Should I create another file in `docs` directory, e.g. `docs/formatter.md`? Maybe we can add JSDoc comments to this `fxp.d.ts` file too, to allow IDEs like VS Code to show inline docs?